.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
```

Comment:
Please add this to the `.gitignore` file

Reply:
this file ensures all the editors behave the same. (i can remove it)